lib/internal/bootstrap/node.js

```diff
@@ -133,6 +133,9 @@ if (!config.noBrowserGlobals) {
   // https://encoding.spec.whatwg.org/#textdecoder
   exposeInterface(global, 'TextDecoder', TextDecoder);
 
+  const { AbortController } = require('internal/abort_controller');
+  exposeInterface(global, 'AbortController', AbortController);
```

Comment:
Without looking at the spec: would it be possible to lazy load this? We load quite a lot of modules during startup by adding this to `globalThis`.

Reply:
For this PR, no, as the actual additional load here is in the event_target PR. I'll open a separate PR to handle that.

Reply:
Would you be so kind and land that PR first in that case? It would otherwise cause many files to be required on startup that are currently not loaded.